src/model_tuner/bootstrapper.py

```diff
@@ -7,6 +7,68 @@
 from random import seed, randint
 
 
+def check_input_type(x):
+    """Method to check input type pandas Series or numpy.
+    Sort index if pandas for sampling efficiency.
+    """
+    # if y is a numpy array cast it to a dataframe
+    if isinstance(x, np.ndarray):
```

Comment:
Need to remove this section from the return_bootstrap_metrics method within the model_tuner as it is now handled in here.

Reply:
sounds good